test/include.test.js

```diff
@@ -53,14 +55,15 @@ describe('include', function() {
 
         u.__cachedRelations.should.have.property('posts');
         u.__cachedRelations.posts.forEach(function(p) {
-          p.userId.should.eql(u.id);
+          p.userId.toString().should.eql(u.id.toString());
```

Comment:
That's a left-over from trying the two designs back and force and there are many other similar cases.  At one point, made all the adhocSort === false cases to use string-based comparison with `should.be.equalOneOf` .

Did the following exercise: Changed `equalOneOf` to `oneOf` (`eql` equivalent) and removed all `toString()` except `toString() is required.